\documentclass[a4paper,11pt,abstracton,hidelinks]{scrartcl}
\usepackage{dphil}
\usepackage{wasysym}
\addbibresource{refs.bib}
% hide section numbers
\setcounter{secnumdepth}{0}


\title{
Chapter 4. Population structure and genetic diversity
}


\author{}


\begin{document}
\renewcommand{\abstractname}{Summary}


\maketitle


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\begin{abstract}


@@


\end{abstract}


\tableofcontents


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Introduction}\label{sec:introduction}


%
In the previous chapter I described the \textit{Anopheles gambiae} 1000 Genomes Project (Ag1000G) phase 1 data resource, which comprises genome variation data from 765 individual mosquitoes sampled from 8 countries spanning sub-Saharan Africa, and includes representation of both \textit{An. gambiae} and \textit{An. coluzzii}.
%
The availability of genomic data from multiple species and geographical locations provides an opportunity to investigate many facets of their population biology and demography.
%
In this chapter I describe analyses of the Ag1000G phase 1 data investigating genetic population structure among the mosquito populations sampled, and characterising levels of genetic diversity within and differentiation between populations.
%
These analyses are interesting because they allow us to begin to build a picture of the underlying demography of these populations, including variations in population size over time and space, and in the degree of connectivity and hence gene flow between populations.
%
Exploring these heterogeneities is particularly relevant because \textit{An. gambiae} and \textit{An. coluzzii} both have an extremely broad geographical and ecological range ~\parencite{dellaTorre2001,TeneFossog2015,Wiebe2017}.
%
\textit{An. coluzzii} is found from the West Coast throughout West and Central Africa.
%
The range of \textit{An. gambiae} overlaps that of \textit{An. coluzzii} and extends across the Great Rift to the East coast, stretching down to South Africa.
%
Both species' ranges span the equator and encompass a remarkably diverse range of environments, including coastal, savanna, sahel and rainforest.
%
Human population density and land use, and the history and current coverage of malaria vector control interventions, also vary substantially throughout this range \parencite{WHO2019WMR}.
%
While we do not have the sampling resolution to attempt to correlate any of these individual variables with genetic features of mosquito populations, we can begin to highlight major variations between populations and generate hypotheses for further investigation.
%


Analyses of population structure and diversity were carried out as part of a broader investigation of population history and demography performed by the Ag1000G Consortium Analysis Working Group.
%
In this chapter I focus on the analyses that I led and performed individually.
%
However, to provide some additional context I also mention some analyses in which I worked together with other Consortium members, and indicate the contributions of others in the relevant sections.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Results}\label{sec:results}


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{The influence of genome architecture on population structure}\label{subsec:treescan}


Investigating genetic population structure means studying the extent to which individual mosquitoes are genetically related to each other, and identifying groups of individuals which are more or less related.
%
The Ag1000G phase 1 resource comprises data on more than 52 million single nucleotide polymorphisms (SNPs) distributed throughout the genome, and thus provides an extremely rich and high-resolution source of information with which to investigate population structure.
%
However, a complication arises because previous genomic studies of \textit{An. gambiae} and \textit{An. coluzzii} have shown that different regions of the genome can convey different information about how individuals are related to each other.
%
In particular, several studies have found that differentiation between the two species \textit{An. gambiae} and \textit{An. coluzzii} is particularly high within certain genome regions but lower or almost absent elsewhere \parencite{Turner2005,White2010,Weetman2012,Cruickshank2014}.
%
There are also a number of large chromosomal inversions that are polymorphic within both of these species~\parencite{dellaTorre2001,Coluzzi2002}.
%
Chromosomal inversions cause a greatly reduced rate of recombination between different karyotypes \parencite{Stump2007}, which in turn will affect patterns of relatedness between individuals.
%
Any analysis of population structure that fails to take these heterogeneities into account will struggle to present a clear picture.
%


\begin{figure}[t!]
\centering
\includegraphics[width=\textwidth]{artwork/chapter4/treescan.pdf}
\caption{Variation across the genome in patterns of relatedness between individual mosquitoes.
%
Panels \textbf{a-d} each show a neighbour-joining tree computed from pairwise genetic distances within a specific 200 kb genomic window, illustrating the four most common patterns of relatedness found throughout the genome.
%
Each node in the tree represents a single individual, coloured by population.
%
\textbf{a}, Example of tree within the 2Rb inversion.
%
\textbf{b}, Example of tree within the 2La inversion.
%
\textbf{a}, Example of tree from euchromatic region of Chromosome 3.
%
\textbf{a}, Example of tree pericentromeric region of the X chromosome.
%
\textbf{e}, Painting of the genome illustrating where the four major patterns of relatedness are found.
%
Each 200 kb window is painted with a colour to indicate which of the four major patterns of relatedness it is most strongly correlated with.
%
}
\label{fig:treescan}
\end{figure}


To explore the relationship between genetic population structure and genome architecture, I divided the genome into non-overlapping 200 kb windows, and then computed pairwise genetic distance between individuals within each window separately.
%
For each window I computed a neighbour-joining tree and visualised each of the resulting trees as an unrooted dendrogram.
%
Several qualitatively different tree topologies were evident in different genome regions (e.g., Fig.~\ref{fig:treescan}a-d).
%
For example, within the pericentromeric region of the X chromosome, trees showed extremely strong clustering by species (e.g., Fig.~\ref{fig:treescan}d).
%
In contrast, trees from euchromatic regions of Chromosome 3 showed some clustering by geographical location, but no clustering by species at all (e.g., Fig.~\ref{fig:treescan}c).
%
Trees within the 2Rb and 2La inversions also had unique topologies, consistent with clustering by inversion karyotype (e.g., Fig.~\ref{fig:treescan}a,b).
%


To analyse these variations in tree topology systematically, I computed the Pearson correlation coefficient between genetic distance matrices from all pairs of genomic windows.
%
I then performed dimensionality reduction on the resulting correlation matrix via multidimensional scaling, to identify common patterns of relatedness found in multiple genome windows.
%
The first three principal coordinates (PCs) from this analysis displayed a strong association with specific patterns of relatedness and genome regions.
%
To visualise these results, I devised a transformation from these first three PCs to different colours representing the different patterns of relatedness, and used these to paint the associated genome windows (Fig.~\ref{fig:treescan}e).
%
The first PC identified the common pattern of relatedness found throughout the 2La inversion.
%
The third PC identified the pattern of relatedness found within the 2Rb inversion.
%
The second PC identified the contrast between the highly species-driven patterns of relatedness found generally in pericentromeric regions, particularly of the X chromosome, and the more geographically-driven patterns found in euchromatic regions of X chromosome and Chromosome 3.
%
There were also a minority of genome windows which did not display a strong correlation any of the four major patterns of relatedness, shown in Fig~\ref{fig:treescan}e as paler colours.
%
These included the windows spanning the insecticide resistance gene \textit{Vgsc} which is found near to the centromere of chromosome arm 2L, and which is known to have experienced adaptive introgression between \textit{An. gambiae} and \textit{An. coluzzii} \parencite{Clarkson2014,Norris2015}.
%
This provides a clue that windows affected by strong positive selection may display unusual patterns of relatedness due to adaptive gene flow between countries and/or species, explored further in Chapter 6.


It is still not clear why we observe such a stark contrast between the pericentromeric and euchromatic regions of the X chromosome, and to a lesser extent Chromosome 3.
%
One factor that undoubtedly plays some part is the reduction in the rate of recombination towards the centromeres \parencite{Cruickshank2014}.
%
Reduced recombination means that selection at linked sites will play a stronger role.
%
This includes purifying selection, which acts to reduce genetic diversity and could accelerate the fixation of different alleles between the two species.
%
In the previous chapter we saw clearly that the level of nucleotide variation was much reduced towards pericentromeric genome regions.
%
This could also include positive selection for speciation genes.
%
The genes involved in maintaining differentiation between \textit{An. gambiae} and \textit{An. coluzzii} still remain a mystery, but if one or more key genes were located towards the pericentromeric region of the X chromosome, then they would be expected to have a substantial effect on the surrounding genome region.
%
\textit{An. gambiae} and \textit{An. coluzzii} hybrids are fully fertile, and the two species are known to undergo some hybridisation in natural populations, the degree of which may vary over both space and time \parencite{Weetman2012,Lee2013}.
%
However, genome regions linked to speciation genes would display very little if any evidence for gene flow between the species due to selection against hybrids, whereas unlinked regions would be much less constrained and alleles could move more freely.
%
In any case, resolving the causes of these contrasting patterns of relatedness is beyond the scope of this thesis, and remains an area of active research and debate.
%
For the present purposes, I continued my investigation of population structure, diversity and differentiation using only the euchromatic regions of Chromosome 3, because this is unaffected by large polymorphic inversions or regions of reduced recombination.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{Population structure}\label{subsec:pop-structure}


To further explore patterns of genetic population structure I used SNPs from euchromatic regions of Chromosome 3 to perform a principal components analysis (PCA)~\parencite{Patterson2006} (Fig.~\ref{fig:pca}).
%
This analysis is a form of dimensionality reduction, and condenses the information present in the hundreds of thousands of SNPs into few principal components (PCs) that explain the maximum amount of variance within the data.
%
In population-genetic terms, each principal component identifies one or more groupings of individuals, such that genetic relatedness is much higher within groupings than between them.
%
PCA in fact has a direct genealogical interpretation, and under certain conditions each principal component will capture a historical split between populations which are separated either geographically or by some other barrier such as assortative mating between species @@cite.
%
In practice a PCA cannot always be interpreted so directly, because other factors such as variations in population size and/or sample size can affect the shape of the result @@cite.
%
Nevertheless, it still allows us to clearly identify genetically distinct populations, by examining how samples group together within the highest PCs.


\begin{figure}[t!]
\centering
\includegraphics[width=\textwidth]{artwork/chapter4/pca.png}
\caption{@@TODO
%
}
\label{fig:pca}
\end{figure}


The first four PCs were clearly elevated above lower PCs in the amount of variance explained, and between them revealed six distinct groupings of individuals.
%
These six groups revealed both geographical and species divisions.
%
Within \textit{An. gambiae}, individuals from Gabon formed a distinct group, and the remaining individuals from Guinea, Burkina Faso, Cameroon and Uganda grouped together.
%
Within \textit{An. coluzzii}, individuals from Burkina Faso and Angola each formed a distinct group.
%
The two remaining groups comprised individuals from Guinea-Bissau and Kenya respectively.
%
The species status of these two groups was uncertain because individuals from Guinea-Bissau displayed a mixture of species genotypes according to conventional molecular assays, and those assays were not performed on the Kenyan samples.
%
I return to the question of species assignment for these groups below.
%
PC5 revealed a further split within \textit{An. gambiae} between Uganda and the remaining individuals.
%
PC6 emphasized the distinction between Burkina Faso \textit{An. coluzzii} and Guinea-Bissau, which were visibly distinct also in higher components but to a lesser degree.
%
PCs 7--10 all displayed some structuring among the Cameroon \textit{An. gambiae}, with individuals from the southern-most collection site spreading out away from the main group of individuals collected from the other more northerly sites.
%


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{Population differentiation}\label{subsec:pop-diff}


To investigate the degree of genetic differentiation between these populations, I computed the pairwise average $F_{ST}$ between all pairs of populations defined by species and country (Fig.~\ref{fig:popdiff}a) (@@cite fst).
%
The $F_{ST}$ statistic summarises the extent to which allele frequencies differ between two groups of individuals, and can be interpreted as an estimate for the degree of genetic drift between the two sampled populations (@@REF).
%
I also computed the rate of doubleton ($f_{2}$) variant sharing between the same pairs of populations (Fig.~\ref{fig:popdiff}b).
%
Doubleton variants are those where the alternate allele is only observed twice, i.e., are rare within the cohort of individuals sampled.
%
In general, doubleton variants are likely to be enriched for recent mutations, because it takes longer time for alleles to reach higher frequencies.
%
Thus patterns of sharing of doubleton variants between individuals are more indicative of recent patterns of relatedness, e.g., can be more sensitive to recent population splits.
%


\begin{figure}[t!]
\centering
\includegraphics[width=1\textwidth]{artwork/chapter4/popdiff.pdf}
\caption{@@TODO
%
}
\label{fig:popdiff}
\end{figure}


These analyses confirmed a strong degree of differentiation between Gabon and other \textit{An. gambiae} populations, with $F_{ST} > 0.051$.
%
In contrast, $F_{ST}$ values between the other four \textit{An. gambiae} populations were at most 0.007.
%
This degree of differentiation was greater than that found between the two species within Burkina Faso, where both species were sampled at the same location ($F_{ST} = 0.031$).
%
The higher $F_{ST}$ values involving Gabon cannot simply be a reflection of greater geographical distance, because the distance between Uganda and \textit{An. gambiae} populations to the west is greater, yet $F_{ST}$ values are much lower.
%
These results indicate that the Gabon individuals are separated by other \textit{An. gambiae} populations to the north by some barrier to gene flow.
%
The obvious candidate for such a barrier is the equatorial rainforest, which represents a major ecological discontinuity.
%
Similarly, \textit{An. coluzzii} from Angola were highly differentiated from \textit{An. coluzzii} from Burkina Faso ($F_{ST} = 0.102$).
%
Again, the level of differentiation was greater than that between the two species within Burkina Faso.
%
Angola is the southern-most sampling location in the Ag1000G phase 1 cohort, and provides further support for a north/south ecogeographical barrier between mosquito populations affecting both species.
%
Of course, further sampling of populations on both sides of the equatorial rainforest, as well as within, will be needed to confirm this hypothesis.


A suprising result was the near-total lack of differentiation between the \textit{An. gambiae} populations from Guinea, Burkina Faso and Cameroon, with $F_{ST}$ reaching at most 0.001 and barely achieving statistical significance (Fig.~\ref{fig:popdiff}a).
%
This lack of differentiation suggests high rates of gene flow between these locations.
%
However, the physical distances are still considerable, with @@ km separating Cameroon from Burkina Faso, and @@ km separating Burkina Faso from Guinea.
%
Previous wisdom is that mosquitoes travel at most 5 km during their lifetime, and therefore expect stronger isolation by distance.
%
Being challenged recently by observation of Anopheles mosquitoes engaging in wind-borne migration, suggesting potential for long-distance migration @@ref.
%
Unfortunately phase 1 doesn't have multiple sites for \textit{An. coluzzii} in West and Central Africa to ask if similarly low levels of structure.
%
Follow up in phase 2 and 3.
%
@@todo edit above para for readability


%
The Kenyan population displayed an extremely high level of differentiation ($F_{ST} > 0.207$) with all other populations, with similar $F_{ST}$ values against populations of both species.
%
This result was somewhat surprising.
%
The Kenyan samples were expected to be genetically isolated to a certain degree, being sampled on the coast at the most easterly location among the collection sites, and being the only samples collected to the east of the Rift Valley, which presents a natural geographical barrier to gene flow.
%
However, only \textit{An. gambiae} and not \textit{An. coluzzii} have been observed previously in this region, thus the Kenyan samples were expected to be \textit{An. gambiae}, which is the reason why conventional molecular diagnostics to differentiation \textit{An. gambiae} from \textit{An. coluzzii} had not been performed.


The analysis of doubleton variants supported all the above observations, with Angola \textit{An. coluzzii} and Gabon \textit{An. gambiae} having very low rates of doubleton sharing with other conspecific populations, consistent with strong isolation.
%
Also provided support for differentiation of Uganda from other \textit{An. gambiae} to the West: despite low $F_{ST}$, Uganda more than twice as likely to share doubleton variants with itself than with any other \textit{An. gambiae}.
%
@@todo any other comments on f2. Kenya, Guinea-Bissau.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{Genetic diversity within populations}\label{subsec:diversity}


\begin{figure}[t!]
\centering
\includegraphics[width=1\textwidth]{artwork/chapter4/pop_params.pdf}
\caption{@@TODO
%
}
\label{fig:diversity}
\end{figure}


In the previous chapter I performed an initial analysis of genetic diversity, computing nucleotide diversity ($\pi$) within the Ag1000G phase 1 cohort as a whole.
%
This analysis of course ignored the fact that different populations within a species may exhibit different levels of genetic diversity, due to differences in their demographic history such as contractions or expansions in population size.
%
To investigate population differences in genetic diversity I performed four further analyses within each of nine populations defined by country and species (Fig. \ref{fig:diversity}).
%
The first of these analyses computed $\pi$, which summarises the fraction of nucleotide differences between pairs of chromosomes within a population (Fig. \ref{fig:diversity}a).
%
The second analyses computed Tajima's $D$, which summarises the distribution of allele frequencies within a population (Fig. \ref{fig:diversity}b).
%
The third analysis computed the full site frequency spectrum (SFS) for each population, which provides information about how many SNPs are observed at different allele frequencies (Fig. \ref{fig:diversity}c).
%
The final analysis examined the decay of linkage disequilibrium, which summarised the degree to which genotypes are correlated at pairs of SNPs at different physical distances from each other (Fig. \ref{fig:diversity}d).
%


These analyses revealed strong contrasts between populations in both the magnitude and architecture of genetic diversity.
%
All the populations to the north of the equatorial rainforest, from Guinea-Bissau in the West to Uganda in the East, including both species, displayed very similar results, with the highest genome-wide average $\pi = 1.5\%$, Tajima's $D < -1.5$, SFS with a strong excess of SNPs at low minor allele frequency and LD decaying to background levels within 2 kb.
%
These characteristics indicate large effective population size and are also consistent with a major population expansion at some point within their history.
%
In both Gabon \textit{An. gambiae} and Angola \textit{An. coluzzii}, $\pi$ was lower, Tajima's $D$ was approaching zero, SFS were more balanced and LD decay was $> 10$ kb.
%
These characteristics are consistent with an effective population size that is both smaller and has been more stable over time.
%
The Kenyan population displayed the most extreme patterns of diversity, with the lowest $\pi$, Tajima's $D > 2$, SFS with a deficit of SNPs at lower minor allele frequencies, and LD not decaying to background until $> 10$ Mb.
%
These characteristics indicate a strong and recent reduction in effective population size.
%
Further examination of levels of heterozygosity within individuals revealed that Kenyan mosquitoes displayed long runs of homozygosity, in some cases affecting almost entire chromosome arms, a pattern not observed in mosquitoes from any other population (Fig. \ref{fig:keroh}).
%
Such patterns of homozygosity are usually only observed after multiple generations of inbreeding, similar to patterns found in domesticated animals for example @@cite.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{Genetic diversity within Cas9 gene drive targets}\label{subsec:gene-drive}


The high level of nucleotide diversity within many of the mosquito populations sampled in Ag1000G phase 1 is interesting in its own right, but also has practical consequences for the development of new vector control tools based on gene drives.
%
Gene drives are a promising future technology for mosquito control, whereby an engineered genetic element is integrated into the mosquito genome that has the capability to be transmitted to progeny with super-Mendelian inheritance @@cite.
%
Because of this biased inheritance, a gene drive will propagate itself within a mosquito population, even if it is in some way deleterious to its carrier.
%
For example, gene drives have been engineered and proven effective under lab conditions which cause population suppression by biasing the sex ratio among offspring to be amlost all male @@cite.
%
Current gene drives make use of CRISPR/Cas9 homing endonucleases, which can be engineered to target almost any gene, because they include a guide RNA which matches a 21 bp sequence in the target genome.
%
The guide RNA can be designed to match any sequence of 18 nucleotides, then must terminate with the protospacer adjacent motif (PAM) \texttt{-NGG}.
%
However, any natural genetic variation in the target sequence within the mosquito population can reduce efficacy of a gene drive, because it will affect binding of the Cas9 guide RNA.
%


\begin{figure}[t!]
\centering
\includegraphics[width=1.1\textwidth,center]{artwork/chapter4/cas9_targets.pdf}
\caption{@@TODO
%
}
\label{fig:cas9}
\end{figure}


I explored the impact of nucleotide variation on the availability of highly conserved Cas9 target sequences, in collaboration with Ag1000G Consortium members Mara Lawniczak and Krzysztof Kozak.
%
I was particularly interested to explore the impact of sample size and of population differences in levels of genetic diversity on the ascertainment of viable Cas9 gene drive target sites.
%
I performed an analysis whereby I identified all 21 bp sequences in the reference genome terminating in a PAM motif, within which no nucleotide variation was found within a given Ag1000G phase 1 population.
%
I repeated this multiple times with successively greater down-sampling of individuals from each population.
%
I then plotted for each population the number of variant sites (Fig. \ref{fig:cas9}a), the number of viable Cas9 targets (Fig. \ref{fig:cas9}b) and the number of genes containing at least one viable Cas9 target (Fig. \ref{fig:cas9}c), and how these metrics varied with increasing sample size.
%
These analyses showed that the higher levels of nucleotide diversity among the populations sampled to the north of the equatorial rainforest had a dramatically greater impact on the reduction of available Cas9 targets and targetable genes, compared with the more southerly populations from Gabon and Angola, and the Kenyan population.
%
At a sample size of 50, more than 1.5 million Cas9 targets remained viable in Kenya, and more than 1 million were viable in Angola and Gabon, whereas less than 0.5 million were viable in any of the remaining populations.
%
Within the whole cohort, less than 3\% of Cas9 targets were fully conserved at the nucleotide level.
%
Clearly some \textit{An. gambiae} and \textit{An. coluzzii} populations will be more naturally resistant to gene drives than others due to higher diversity, and will require more intensive sampling in order to survey existing variation and design gene drives to accommodate this.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\subsection{Populations with uncertain species status}\label{subsec:species}


As mentioned above, the species assignment for individuals from Guinea-Bissau and Kenya was uncertain.
%
The results of conventional molecular assays for differentiating \textit{An. gambiae} and \textit{An. coluzzii} @@cite showed a mixture of genotypes in Guinea-Bissau, with some individuals homozygous for the \textit{An. gambiae} marker allele, some homozygous for the \textit{An. coluzzii} marker allele, and some heterozygous.
%
Several previous studies have observed similar results from sampling mosquitoes in coastal areas of the far West of Africa, and have interpreted these results as evidence for a localised breakdown in reproductive isolation between \textit{An. gambiae} and \textit{An. coluzzii} \parencite{Oliveira2008,Marsden2011,Weetman2012,Gordicho2014,Vicente2017}.
%
The conventional molecular diagnostic assays only a single locus on the X chromosome, and Ag1000G Consortium member Giordano Botta used a previous study of \textit{An. gambiae} and \textit{An. coluzzii} in Mali to ascertain a set of @@N SNPs distributed across all chromosomes that were highly differentiated between the two species and could be used as ancestry-informative markers (AIMs) to explore species ancestry across the genome in the Ag1000G phase 1 cohort.
%
Genotypes at these AIMs confirmed that Guinea-Bissau individuals carried an apparent mixture of \textit{An. gambiae} and \textit{An. coluzzii} alleles on all chromosome arms @@cite.
%
However, this apparent mixture of species ancestry was somewhat puzzling given the results of population structure analyses shown above.
%
For example, in PCA the Guinea-Bissau individuals form a single grouping (\ref{fig:pca}).
%
If these individuals were sampled from a population resulting from ongoing hybridisation between \textit{An. gambiae} and \textit{An. coluzzii} then we would expect to see some structure within these individuals.
%
In particular, we would expect the Guinea-Bissau individuals to appear spread out on the PCA between the West African \textit{An. gambiae} and \textit{An. coluzzii} populations, reflecting varying degrees of admixture.

%
To explore evidence for admixture within Guinea-Bissau I performed allele frequency tests for admixture using the $f_{3}$ and $f_{4}$ statistics defined by @@cite.
%
The $f_{3}$ statistic tests whether allele frequencies in a putatively admixed population are on average intermediate between to populations that are representative of the donor populations.
%
@@TODO f3 results.
%
The $f_{4}$ statistic tests for admixture by examining whether there is any bias in allele sharing between @@what.
%
@@TODO f4 results.
%
@@TODO Conclude what do these tests show. Other possible hypotheses.


A surprising finding from the AIM analysis was that individuals from Kenya also carried an apparent mixture of \textit{An. gambiae} and \textit{An. coluzzii} alleles on all chromosome arms @@cite.
%
This is unexpected because no mosquitoes typing as \textit{An. coluzzii} via conventional molecular diagnostics have ever been found in East Africa, and hence these assays were not even performed on the Kenyan mosquitoes.
%
Given the absence of \textit{An. coluzzii} from this geographical region, we can rule out any recent admixture between species as the cause.
%
This leaves ancient admixture and ancestral population as possible explanations.
%
@@TODO f3 results
%
@@TODO f4 results.
%
@@TODO what do they say about Kenya.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Discussion}\label{sec:discussion}


@@TODO para about low differentiation between gambiae in west/central Africa. suggests high rates of gene flow. Followed up in collaboration with members of the consortium, used joint site frequency spectra to model and fit migration rate. Although still crude, was at upper bound, indicating apparently panmictic.


@@TODO Another interesting feature of these populations is highest diversity with excess of rare variants. Followed up with consortium to fit population size history using SFS via dadi and stairway plot. All strongly supported large population growth on historical timescale. Hard to estimate but timing in the range @@. Open question regarding which variants to use, used intergenic but in hindsight perhaps better to use syn sites, twice diversity would double pop size estimates, also selection might alter shape of SFS thus timing of inferred expansions.


@@TODO para about high differentiation across equatorial rainforest, replicated within both gambiae and coluzzii. Ref back to pinto and lehmann studies. Suggests what.


@@TODO para about Kenya very unusual. Low diversity and runs of homozygosity suggest recent bottleneck. Followed up with consortium, used IBD to estimate recent population history. Provided strong support for bottleneck, however timing hard to resolve, appeared to predate the bednet scaleup but clearly some limitations to method. Other unusual feature of this population is uncertain species status. Needs replication but outside range of coluzzii and so cannot be recent hybrid. Further analyses and sampling needed to resolve if cryptic taxa or some other kind of hybrid.


@@TODO para about guinea-bissau, species status, is it recent breakdown of reproductive isolation or not? pull together evidence from PCA, f3, f4, suggest more complex than that. Clearly a coherent population. Remaining hypotheses, more ancient admixture, or possibly some other kind of split. Important for malaria control because acts like a distinct taxon. Futher sampling needed to understand contemporary patterns of gene flow and isolation.


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Methods}\label{sec:methods}


@@


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Conclusion}\label{sec:conclusion}


@@


%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Acknowledgments}\label{sec:acknowledgments}


@@


\printbibliography


\clearpage
\beginsupplement
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Supplemental figures}\label{sec:supplemental-figures}


\begin{figure}[h!]
\centering
\includegraphics[width=1\textwidth]{artwork/chapter4/keroh.pdf}
\caption{@@TODO
%
}
\label{fig:keroh}
\end{figure}



@@


\clearpage
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\section{Supplemental tables}\label{sec:supplemental-tables}


@@


\end{document}
